packages/components/inputs/rich-text-utils/src/rich-text-body/rich-text-body.styles.ts

```diff
@@ -2,7 +2,14 @@ import styled from '@emotion/styled';
 import { css } from '@emotion/react';
 import { customProperties as vars } from '@commercetools-uikit/design-system';
 
-const getBorderColor = (props) => {
+type Props = {
```

Comment:
Maybe we can pick props from the `TRichTextEditorBody` type defined in the `rich-text-body.tsx` file and name this one like `TRichTextBodyStylesProps`.
What do you think about it?